data/osx/Cataclysm.sh

```diff
@@ -4,6 +4,7 @@ cd "${PWD}/../Resources/"
 
 COMMAND='
 OSREV=`uname -r | cut -d. -f1`;
+export TERMINFO=/usr/share/terminfo;
```

Comment:
There is no need to export `TERMINFO=/usr/share/terminfo;`. According to the apple [terminfo manual](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man5/terminfo.5.html):

> If neither TERMINFO nor TERMINFO_DIRS is set, the last place tried will be the system terminfo directory, directorytory, /usr/share/terminfo.